tests/phpunit/tests/Discovery.php

```diff
@@ -59,16 +59,7 @@ public static function wpTearDownAfterClass() {
 
 	public function setUp() {
 		parent::setUp();
-		global $wp_query, $post;
-		$post                     = get_post( self::$story_id );
-		$wp_query->queried_object = $post;
-	}
-
-	public function tearDown() {
-		global $wp_query, $post;
-		unset( $wp_query->queried_object );
-		unset( $post );
-		parent::tearDown();
+		$this->go_to( get_permalink( self::$story_id ) )
```

Comment:
Missing semicolon causes syntax error:

```suggestion
		$this->go_to( get_permalink( self::$story_id ) );
```

Reply:
Fixed already.